internal/restic/node.go

```diff
@@ -188,8 +188,6 @@ func (node Node) restoreMetadata(path string) error {
 	if err := lchown(path, int(node.UID), int(node.GID)); err != nil {
 		// Like "cp -a" and "rsync -a" do, we only report lchown permission errors
 		// if we run as root.
-		// On Windows, Geteuid always returns -1, and we always report lchown
```

Comment:
What is the reason why this comment was removed?

Reply:
Because this is dead code on Windows; lchown is a stub that returns nil.